pom.xml

```diff
@@ -96,6 +96,7 @@
         <ddf-branding>DDF</ddf-branding>
         <ddf-branding-lowercase>ddf</ddf-branding-lowercase>
         <ddf-branding-expanded>Distributed Data Framework</ddf-branding-expanded>
+        <ddf.version>${project.version}</ddf.version>
         <!--Needed for documentation formatting-->
         <at-symbol>@</at-symbol>
```

Comment:
having a literal `@` symbol will escape the rest of the substitutions on the same line. It's a workaround.